server/db/utils/user_crewHelpers.js

```diff
@@ -43,12 +43,14 @@ exports.getCrewMembers = (crewId, cb) => {
     });
 };
```

Comment:
it actually does work the old way since the inputs were fixed, but if this works too I'm fine keeping it this way. Just adds more lines of code.